extensions/positron-r/src/commands.ts

```diff
@@ -126,6 +126,13 @@ export async function registerCommands(context: vscode.ExtensionContext, runtime
 			}
 		}),
 
+		vscode.commands.registerCommand('r.packageTestExplorer', async () => {
+			vscode.commands.executeCommand('workbench.view.testing.focus');
+			// Wait for the test explorer to set up before running tests
+			await delay(500);
+			vscode.commands.executeCommand('testing.runAll');
```

Comment:
- We don't need to check if R is up, devtools is installed, etc, because the R extension's Test Explorer support already does this.
- I did not map straight from the keybinding to `'testing.runAll'` because it doesn't focus the Test Explorer and needs to have the Test Explorer already set up. 
- This will _not_ respect `TESTTHAT_MAX_FAILS` (i.e. related to #2723) because these tests are run in separate processes IIUC; it wouldn't do us any good to pass this env var through to the test runner. We would need to follow up with additional work to have this approach for running tests respect `TESTTHAT_MAX_FAILS`.